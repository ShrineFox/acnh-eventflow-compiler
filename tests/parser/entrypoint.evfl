flow Test():
    TestActor1.Action0()
entrypoint EP1:
    pass
entrypoint EP2:
    TestActor2.Action0()
entrypoint EP3:
    return

flow Test2():
    TestActor1.Action0()

