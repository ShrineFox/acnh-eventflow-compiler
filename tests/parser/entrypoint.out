RootNode[name=EP1, vardefs=[], out_edges=[Event3]]
RootNode[name=EP2, vardefs=[], out_edges=[Event3]]
RootNode[name=EP3, vardefs=[], out_edges=[]]
RootNode[name=Test, vardefs=[], out_edges=[Event0]]
RootNode[name=Test2, vardefs=[], out_edges=[Event7]]
ActionNode[name=Event0, action=EventFlowActionAction0(), params={}, out_edges=[Event3]]
ActionNode[name=Event3, action=EventFlowActionAction0(), params={}, out_edges=[]]
ActionNode[name=Event7, action=EventFlowActionAction0(), params={}, out_edges=[]]

